examples/extern_methods.nit

```diff
@@ -54,16 +62,25 @@ redef enum Int
 		printf("from C: self + fib(self) = %s\n", c_string);
 	`}
 
-	# Equivalent to foo but written in pure Nit
+	# Equivalent to `foo` but written in pure Nit
 	fun bar do print "from Nit: self + fib(self) = {self+self.fib}"
 end
 
+redef class Float
+	# Arctangent of `self` and `other` using `atan2` from libmath
```

Comment:
the usual name for the C math library is `math.h`. So "using `atan2` from `math.h`"